Algorithms/Graphs/MST/prim.cpp

```diff
@@ -0,0 +1,90 @@
+/* The following program represents the Prim's algorithm
+for finding the Minimum Spanning Tree of a graph represented
+as an adjacency matrix (2d matrix).
+
+The time complexity of this algorithm is O(v^2), where
+v is the number of vertices, since adjacent vertices of each
+vertex are being iterated through (v*v).
+
+*/
+
+#include <iostream>
+#include<bits/stdc++.h>
+
+using namespace std;
+
+int findMin( int* keys, bool* included, int v){
+    int mini = INT_MAX;
+    int index;
+    int i;
+
+    for( i = 0; i < v; i ++){
+        if( !included[i] && keys[i] < mini){
+            mini = keys[i];
+            index = i;
+        }
+    }
+    return index;
+}
+
+int main(){
+    cout<<"Enter number of vertices of the graph"<<endl;
+    int v;
+    cin>>v;
+
+    int graph[v][v];
+    int i, j; //Iterators
+
+    cout<<"Enter number of edges of the graph"<<endl;
+    int e;
+    cin>>e;
+
+    cout<<"Enter the vertices and their edge weights in the following format :"<<endl;
+    cout<<"Vertex1 Vertex2 Weight"<<endl;
+    cout<<"PS : First vertex should be 0."<<endl;
+
+    while(e--){
```

Comment:
Never modify the number of edges variable. What if you need it for other functions/purposes later? Either use a for loop or store the value so you always have the number of edges retained.